test/SimpleApp.DeviceTests/SimpleApp.DeviceTests.csproj

```diff
@@ -54,6 +58,58 @@
     <PackageReference Include="DeviceRunners.XHarness.Maui" Version="0.1.0-preview.2" />
     <PackageReference Include="DeviceRunners.XHarness.Xunit" Version="0.1.0-preview.2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Sentry">
+      <HintPath>..\..\sentry\net8.0-android\android-arm64\Sentry.dll</HintPath>
+    </Reference>
+    <Reference Include="Sentry.Android.AssemblyReader">
+      <HintPath>..\..\sentry\net8.0-android\android-arm64\Sentry.Android.AssemblyReader.dll</HintPath>
+    </Reference>
```

Comment:
Do we need Sentry in this case?

I imagine we'd just need a DeviceTest thing set up so you can write C# and run in the device.
Then load CastleProxy (or NSubstitute/Moq) as a dependency and do something with it similar to when it fails on our environment

Reply:
That's what this was supposed to be:

https://github.com/jamescrosswell/net8-mock-internal/blob/b9abf8e92aaf7537c26674100e0dfe0d46d60741/test/SimpleApp.DeviceTests/Tests/StrongLibTests.cs#L15-L27

The theory was that what was tripping it up was resolving internal members of a strongly named assembly on iOS/Android. However, unfortunately, the above test passes on Android... whereas the Sentry tests that use NSubstitute fail. So the only way I can think to reproduce the issue is to use the Sentry tests.